### ISSUE TITLE
unreliable test: "WinstonToolkitLogger logs to a file logs info"

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-toolkit-vscode#2095]
```
  1) WinstonToolkitLogger
       logs to a file
         logs info:

      AssertionError [ERR_ASSERTION]: Expected log message was missing
      + expected - actual

      -false
      +true
      
      at Suite.<anonymous> (d:\a\aws-toolkit-vscode\aws-toolkit-vscode\src\test\shared\logger\winstonToolkitLogger.test.ts:152:24)
      at Generator.next (<anonymous>)
      at fulfilled (d:\a\aws-toolkit-vscode\aws-toolkit-vscode\dist\src\test\shared\logger\winstonToolkitLogger.test.js:9:58)
```

**Comment by JadenSimon** [comment]
I have a PR for this but I'll sit on it until we can clear out some outstanding PRs.

**Comment by justinmk3** [comment]
Fixed by #2397
